maskwatch-bot/__init__.py

```diff
@@ -294,10 +295,16 @@ def _mask_format(self,
             mask:    str,
             details: str
             ) -> str:
+
+        last_hit = ""
+        if details.last_hit is not None:
+            last_hit = to_pretty_time(int(time()-details.last_hit))
+            last_hit = f", {last_hit} ago"
```

Comment:
I would probably want to say something like `, last hit {last_hit} ago` instead of just `, {last_hit} ago` as someone not familiar with the bot might be confused by that.

Reply:
I intentionally left that off for the sake of line length, but on second thought it's probably worth the space cost.